src/plugins/apps/index.js

```diff
@@ -0,0 +1,65 @@
+//@ts-check
+const { default: fetch } = require('node-fetch');
+
+module.exports = async function appsPlugin() {
+  // TODO: actually use the electron/apps repo as a data source
+  const response = await fetch(
+    'https://raw.githubusercontent.com/erickzhao/apps/master/index.json'
```

Comment:
Do we need to wait for you to submit a PR to electron/apps and merge it before merging this PR?